src/components/dashboard/ActionsList.js

```diff
@@ -0,0 +1,196 @@
+         word-break: break-all;
+      }
+
+      .dropdown-image-right {
+         width: 10px;
+         margin: 0 0 0 0;
+      }
+      .dropdown-image {
+         float: right;
+      }
+
+      &.dropdown-down {
+         background-color: #f8f8f8;
+
+         .dropdown-image-right {
+            width: 10px;
+            top: 0px;
+            left: 12px;
+         }
+      }
+
+      &.showsub {
+         .dropdown-image-right {
+            left: -24px;
+         }
+      }
+      &.showsub.dropdown-down {
+         .dropdown-image-right {
+            left: -6px;
+         }
+      }
+
+      @media screen and (max-width: 767px) {
+         &.showsub {
+            .dropdown-image-right {
+               left: -14px;
+            }
+         }
+         &.showsub.dropdown-down {
+            .dropdown-image-right {
+               left: 4px;
+            }
+         }
+
+         .main-row-title {
+            a {
+               font-size: 14px;
+            }
+         }
+      }
+   }
+`
+
+const ActionsList = ({ transaction, actions, wide }) => 
+   actions
+      .slice(0,5)
+      .map((a, i) => (
+         <ActionRow 
+            key={`action-${i}`} 
+            transaction={transaction} 
+            action={a} 
+            actionKind={Object.keys(a)[0]}  
+            wide={wide}
+            i={i}
+         />
+))
+
+const ActionRow = ({ transaction, action, actionKind, wide, showSub = false, toggleShowSub, showSubOpen, i }) => (
+   <CustomGridRow
+      verticalAlign='middle'
+      className={`${wide ? `wide` : ``} ${
+         showSub && showSubOpen === i ? `dropdown-down` : ``
+      } ${showSub ? `showsub` : ``}`}
+      onClick={() => wide && toggleShowSub(i, action)}
+   >
+      <Grid.Column
+         computer={wide ? 15 : 16}
+         tablet={wide ? 14 : 16}
+         mobile={wide ? 14 : 16}
+      >
+         <Grid verticalAlign='middle'>
+            <Grid.Column className='col-image'>
+               <ActionIcon actionKind={actionKind} />
+            </Grid.Column>
+            <Grid.Column className='main-row-title color-black border-bottom'>
+               <ActionMessage 
+                  transaction={transaction}
+                  action={action}
+                  actionKind={actionKind}
+               />
+            </Grid.Column>
+         </Grid>
+      </Grid.Column>
+      {wide && (
+         <Grid.Column
+            computer={1}
+            tablet={2}
+            mobile={2}
+            textAlign='right'
+         >
+            <Image
+               src={showSub && showSubOpen === i ? ArrowBlkImage : ArrowRight}
+               className='dropdown-image dropdown-image-right'
+            />
+            {/* <span className='font-small'>{row[3]}</span> */}
+         </Grid.Column>
+      )}
+   </CustomGridRow>
+)
+
+const ActionMessage = ({ transaction, action, actionKind }) => (
+   <Fragment>
+      {actionKind === 'createAccount' && `New account created: ${transaction.receiverId}`}
```

Comment:
Let's make sure new stuff is localizable https://github.com/nearprotocol/near-wallet/issues/217